README.md

```diff
@@ -3,22 +3,23 @@
 
 [![Open in
 Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/datamindedacademy/academy_dbt.git)
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/datamindedacademy/academy_dbt)
 
 This repository is hosting the exercises provided to students in the context of the dbt course of the Data Minded Academy.
 
-To start click on "Open in Gitpod" button.
```

Comment:
Since you remove this comment, should the badge to gitpod also be removed? Or you want to keep it around, just in case?

Reply:
I did want to keep around the "open in gitpod" as a backup, but I think I'll move it to the bottom of the readme, to minimize confusion.